tests/test_suite/test_utils.py

```diff
@@ -0,0 +1,27 @@
+import subprocess, logging, re
+
+LOGGER = logging.getLogger(__name__)
+
+def get_ansible_response_status(ansible_response):
```

Comment:
whilst trying to verify this, I get an error:

```
E       AttributeError: 'NoneType' object has no attribute 'group'
```
as the `ansible-response` is  just an empty string --`''`

Reply:
Hi @dimakis how did you run it? I can not reproduce your issue.
I recommend following the steps I described in the README file: https://github.com/agullon/app-services-ansible/tree/add_test_framework#how-to-execute-tests